opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchQueryRequest.java

```diff
@@ -151,7 +153,8 @@ public OpenSearchQueryRequest(StreamInput in, OpenSearchStorageEngine engine) th
     }
 
     OpenSearchIndex index = (OpenSearchIndex) engine.getTable(null, indexName.toString());
-    exprValueFactory = new OpenSearchExprValueFactory(index.getFieldOpenSearchTypes());
+    exprValueFactory =
+        new OpenSearchExprValueFactory(index.getFieldOpenSearchTypes(), fieldTypeTolerance);
```

Comment:
Logic should similar to OpenSearchScrollRequest right? If yes, fieldTypeTolerance should not associate with OpenSearchQueryRequest.

```
    OpenSearchIndex index = (OpenSearchIndex) engine.getTable(null, indexName.toString());
    exprValueFactory =
        new OpenSearchExprValueFactory(
            index.getFieldOpenSearchTypes(), index.isFieldTypeTolerance());
```

Reply:
maybe another option is to create `OpenSearchExprValueFactory` in request builder and pass it to request? In case factory needs more settings passed in future?

Reply:
Updated.